docs/source/user-guide/general/drift_guide.rst

```diff
@@ -0,0 +1,301 @@
+
+In computer vision specifically, we can't measure drift on images directly, as the individual pixel has little
+value when estimating drift. Also, labels in computer vision are sometimes complex structures as well (for example, in
+object detection, an image can have any number of bounding boxes).
+Therefore, the computer vision checks use :doc:`image and label properties</user-guide/vision/vision_properties>` to estimate
+drift, as image data and labels are not simple one-dimensional variables.
+
+To detect `data <#data-drift>`__ or `concept drift <#concept-drift>`__, deepchecks offers the
+:doc:`Image Property Drift check </checks_gallery/vision/train_test_validation/plot_image_property_drift>` which uses
+`univariate measures <#detection-by-univariate-measure>`__ and the :doc:`Image Dataset Drift check</checks_gallery/vision/train_test_validation/plot_image_dataset_drift>`
+which uses a `domain classifier <#detection-by-domain-classifier>`__ in order to detect multivariate drift.
+
+For drift in your label's distribution, deepchecks offers the :doc:`Label Drift check </checks_gallery/vision/train_test_validation/plot_train_test_label_drift>`,
+which also uses `univariate measures <#detection-by-univariate-measure>`__.
+
+In cases where the label is not available, we strongly recommend to also use the :doc:`Prediction Drift check</checks_gallery/vision/model_evaluation/plot_train_test_prediction_drift>`,
+which uses the same methods but on the model's predictions, and can detect possible changes in the distribution of the label.
+
+For code examples, see `here <#computer-vision-checks>`__
+
+What Can You Do in Case of Drift?
+=================================
+
+.. image:: /_static/images/general/deepchecks_label_drift.png
+   :alt: Output of the TrainTestLabelDrift check
+   :align: center
+
+When suspecting drift in your data, you must first understand what changed in the data - were it the features, the labels,
+or maybe just the predictions.
+In deepchecks, we show you a drift score and the drift on your most :doc:`important features </user-guide/tabular/feature_importance>`,
+giving you an insight on the severity of your drift, even if your not still sure of its source.
+
+It is recommended to manually explore your data and try to understand the root cause of your changes, in order to
+estimate the effect of the change on your model's performance.
+After you have deeper insights on your data, you can choose to act in one of the following ways:
+
+
+Retrain Your Model
+^^^^^^^^^^^^^^^^^^
+
+If you have either kind of drift, retraining your model on new data that better represents the current distribution,
+is the most straight-forward solution.
+However, this solution may require additional resources such as manual labeling of new data, or might not be possible
+if labels on the newer data are not available yet.
+
+Retraining is usually necessary in cases of concept drift. However, retraining may still be of use even for label drift
```

Comment:
A bit weird IMO that label drift is mentioned here for kinda the first time as a phenomenon (and not a check)